[2022-08-15 20:50:32,441] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETL_DAG.Load_into_price_index_table manual__2022-08-15T20:40:50.207703+00:00 [queued]>
[2022-08-15 20:50:32,459] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETL_DAG.Load_into_price_index_table manual__2022-08-15T20:40:50.207703+00:00 [queued]>
[2022-08-15 20:50:32,461] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-15 20:50:32,462] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-15 20:50:32,463] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-15 20:50:32,490] {taskinstance.py:1397} INFO - Executing <Task(S3ToSnowflakeOperator): Load_into_price_index_table> on 2022-08-15 20:40:50.207703+00:00
[2022-08-15 20:50:32,499] {standard_task_runner.py:52} INFO - Started process 1104 to run task
[2022-08-15 20:50:32,505] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_DAG', 'Load_into_price_index_table', 'manual__2022-08-15T20:40:50.207703+00:00', '--job-id', '1041', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpcoucuyyt', '--error-file', '/tmp/tmp3lm49cjy']
[2022-08-15 20:50:32,508] {standard_task_runner.py:80} INFO - Job 1041: Subtask Load_into_price_index_table
[2022-08-15 20:50:32,624] {task_command.py:371} INFO - Running <TaskInstance: ETL_DAG.Load_into_price_index_table manual__2022-08-15T20:40:50.207703+00:00 [running]> on host 42abdd2e54ec
[2022-08-15 20:50:32,708] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=narapady
AIRFLOW_CTX_DAG_ID=ETL_DAG
AIRFLOW_CTX_TASK_ID=Load_into_price_index_table
AIRFLOW_CTX_EXECUTION_DATE=2022-08-15T20:40:50.207703+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-15T20:40:50.207703+00:00
[2022-08-15 20:50:32,710] {s3_to_snowflake.py:135} INFO - Executing COPY command...
[2022-08-15 20:50:32,734] {base.py:68} INFO - Using connection ID 'snowflake_conn' for task execution.
[2022-08-15 20:50:32,739] {connection.py:275} INFO - Snowflake Connector for Python Version: 2.7.11, Python Version: 3.7.13, Platform: Linux-5.10.104-linuxkit-aarch64-with-debian-11.3
[2022-08-15 20:50:32,742] {connection.py:927} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2022-08-15 20:50:32,744] {connection.py:944} INFO - Setting use_openssl_only mode to False
[2022-08-15 20:50:33,777] {cursor.py:715} INFO - query: [ALTER SESSION SET autocommit=True]
[2022-08-15 20:50:34,051] {cursor.py:739} INFO - query execution done
[2022-08-15 20:50:34,053] {snowflake.py:328} INFO - Running statement: COPY INTO public.price_index
FROM @s3_stage/
files=('price-index-staged.csv')
file_format=csv_format, parameters: None
[2022-08-15 20:50:34,054] {cursor.py:715} INFO - query: [COPY INTO public.price_index FROM @s3_stage/ files=('price-index-staged.csv') fi...]
[2022-08-15 20:50:35,913] {cursor.py:739} INFO - query execution done
[2022-08-15 20:50:35,915] {snowflake.py:338} INFO - Statement execution info - {'file': 's3://s3-bucket-staged/price-index-staged.csv', 'status': 'LOADED', 'rows_parsed': 38, 'rows_loaded': 38, 'error_limit': 1, 'errors_seen': 0, 'first_error': None, 'first_error_line': None, 'first_error_character': None, 'first_error_column_name': None}
[2022-08-15 20:50:35,916] {snowflake.py:342} INFO - Rows affected: 1
[2022-08-15 20:50:35,917] {snowflake.py:343} INFO - Snowflake query id: 01a64fc2-0004-28a4-001e-000700041116
[2022-08-15 20:50:35,919] {connection.py:557} INFO - closed
[2022-08-15 20:50:36,048] {connection.py:560} INFO - No async queries seem to be running, deleting session
[2022-08-15 20:50:36,327] {s3_to_snowflake.py:137} INFO - COPY command completed
[2022-08-15 20:50:36,362] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=ETL_DAG, task_id=Load_into_price_index_table, execution_date=20220815T204050, start_date=20220815T205032, end_date=20220815T205036
[2022-08-15 20:50:36,409] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-15 20:50:36,438] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
